tsconfig.json

```diff
@@ -4,6 +4,13 @@
 		"declaration": false,
 		"emitDecoratorMetadata": true,
 		"experimentalDecorators": true,
+		"lib": [
+			"dom",
+			"es5",
+			"es2015.iterable",
+			"es2015.symbol",
+			"es2015.symbol.wellknown"
+		],
```

Comment:
@dylans just to let you know this has already been done as part of the changes for the reporter.